pkg/api/validation/validation.go

```diff
@@ -441,9 +441,27 @@ func validateHostPathVolumeSource(hostPath *api.HostPathVolumeSource) errs.Valid
 
 func validateGitRepoVolumeSource(gitRepo *api.GitRepoVolumeSource) errs.ValidationErrorList {
 	allErrs := errs.ValidationErrorList{}
+	target := gitRepo.Directory
+
 	if gitRepo.Repository == "" {
 		allErrs = append(allErrs, errs.NewFieldRequired("repository"))
 	}
+	if path.IsAbs(target) {
```

Comment:
Oh @pmorie  Sorry, I misunderstood your words. I though you mean to update ALL path validations in validated.go

Will be updated in this pr :)